src/atoms/BorderedButton.js

```diff
@@ -17,18 +23,14 @@ const BorderedButton = Button.extend`
 	font-weight: 500;
 	letter-spacing: 0;
 	outline: none;
+	cursor: pointer;
```

Comment:
Isn't cursor: pointer default for buttons?

Reply:
nope

Reply:
ok :)